static/elements/form-field-specs.js

```diff
@@ -454,20 +480,24 @@ export const ALL_FIELDS = {
   'i2e_lgtms': {
     help_text: html`
       Full email address of API owner who LGTM\'d the 
-      Intent to Experiment email thread.`,
+      Intent to Experiment email thread. 
+      This field fills automatically
```

Comment:
Sadly, that part of internals/detect_intent.py does not work.  So, I think it would be best to leave this new sentence off.  Eventually, we'll phase out this field.

Reply:
OK.